### ISSUE TITLE
any ideas?

### ISSUE THREAD

**Issue by trace7087** [trueToastedCode/ParallelsLab#11]
I can't seem to get this to work, any ideas why not please?
![Uploading Screenshot 2023-11-06 at 18.22.59.png…]()

**Comment by trace7087** [comment]
if anyone else comes across this issue please make sure that you don't already have Xcode tools installed via the command line as this only seems to work when you download it from apple apps. if you do then you have to delete all files and folders that associate with Xcode. 
![Uploading Screenshot 2023-11-06 at 18.55.27.png…]()

**Comment by trueToastedCode** [comment]
> hat associate with Xcode.

This seems related to #5...

**Comment by royalts352** [comment]
How to block the P Servers as last steps on a Mac?

**Comment by trueToastedCode** [comment]
> How to block the P Servers as last steps on a Mac?

`sudo nano /etc/hosts`, then copy-paste the entries and lock using `sudo chflags uchg /etc/hosts && sudo chflags schg /etc/hosts`

**Comment by royalts352** [comment]
> > How to block the P Servers as last steps on a Mac?
> 
> `sudo nano /etc/hosts`, then copy-paste the entries and lock using `sudo chflags uchg /etc/hosts && sudo chflags schg /etc/hosts`

He thank you for your fast response and I tryed it  the whole day to fix this but I don't get access to my hosts file. I disabled the SIP I granted the full disk access and so on but I can't get the locked symbol on this file like u see in the screenshot. I googled so much but no-one has my exact problem.
But on my defense I need to tell I am pretty new to Mac and the os system.

I would be glad if you had any idea I could google for.

this makes me ....

![Bild 07 11 23 um 15 59](https://github.com/trueToastedCode/ParallelsLab/assets/47079390/bcf525b1-3f12-4f28-b3d4-984d173dbe8e)

Thank you for your work
